cmake/projects/YAML_CPP/hunter.cmake

```diff
@@ -0,0 +1,21 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME
+  YAML_CPP
```

Comment:
> I think it would be better to rename the project to yaml-cpp and rewrite the package project name also to this

Will you do it? Should I pause merge? I'm okay with merging it as is, though I think `YAML_CPP` is a strange name :)